docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+  * 6.4 [Choose a personName](#6-4-choose-a-personname)
+  * 6.5 [Choose a namePattern](#6-5-choose-a-namepattern)
+  * 6.6 [Examples of choosing a namePattern](#6-6-examples-of-choosing-a-namepattern)
+    * 6.6.1 [Examples for rules 1 and 2](#examples-for-rules-1-and-2)
+    * 6.6.2 [Examples for rule 3 and the interaction between the rules](#examples-for-rule-3-and-the-interaction-between-the-rules)
+  * 6.7 [Deriving initials](#6-7-deriving-initials)
+  * 6.8 [Handling foreign names](#6-8-handling-foreign-names)
+* 7 [Sample Name](#7-sample-name)
+  * 7.1 [Syntax](#7-1-syntax)
+  * 7.2 [Expected values](#7-2-expected-values)
+* 8 [PersonName Data Interface Examples](#8-personname-data-interface-examples)
 
-## 1 <a name="AaAaaa" href="#AaAaaa">Aa Aaaa</a>
+## 1 <a name="CLDRPersonNames" href="#CLDRPersonNames">CLDR Person Names</a>
 
-Section 1 body text xxxxx.
+### 1.1 <a name="Introduction" href="#Introduction">Introduction</a>
 
-### 1.1 <a name="BbBbbb" href="#BbBbbb">Bb Bbbb</a>
+CLDR provides formatting for person names, such as John Smith or 宮崎駿. These use patterns to show how a name record (for example, from a database) should be formatted for a particular locale. Name data has fields for the parts of people’s names, such as a **given **field with a value of “Maria”, and a **surname **field value of “Schmidt”. 
 
-Section 1.1 body text xxxxx.
+There is a wide variety in the way that people’s names appear in different languages. 
 
-## 2 <a name="CcCccc" href="#CcCccc">Cc Cccc</a>
+* People may have a different number of names, depending on their culture—they might have only one name (“Zendaya”), two (“Albert Einstein”), or three or more.
+* People may have multiple words in a particular name field, eg “Mary Beth” as a given name, or “van Berg” as a surname.
+* Some languages, such as Spanish, have two surnames (where each can be composed of multiple words).
+* The ordering of name fields can be different across languages, as well as the spacing (or lack thereof) and punctuation.
+* Name formatting needs to be adapted to different circumstances, such as a need to be presented shorter or longer; formal or informal context; or when talking about someone, or talking to someone, or as a monogram (JFK).
 
-Section 2 body text xxxxx.
+This document provides the [LDML](http://www.unicode.org/reports/tr35/) specification for formatting of personal names, using data, structure, and examples. 
 
-### 2.1 <a name="DdDddd" href="#DdDddd">Dd Cddd</a>
+    **This is a technology preview; thus not intended for production software (except where itself marked as a technology preview). We have gathered a first round of data, and intend to refine the way in which we gather data. We are looking for additional feedback on the tech preview so that we can make improvements. For example, there are a few areas where we intend enhancements: handling native vs foreign names (in the native script); handling prefix and suffix fields; and so on.**
```

Comment:
for block quotes the following works https://github.github.com/gfm/#block-quotes

Reply:
Changed to use ">"